### ISSUE TITLE
TypeError: Cannot read property 'get' of undefined

### ISSUE THREAD

**Issue by ferco0** [L2jLiga/fastify-decorators#22]
When inject a service in controller using the decorator

How to reproduce: 
```typescript
import { Controller, GET, Inject } from "fastify-decorators";
import { AnyService } from "../services";

@Controller("/status")
export default class HelloController {
  @Inject(AnyService)
  private anyService: AnyService;

  @GET()
  async handler(): Promise<{ hello: string }> {
    try {
      console.log(this.anyService);
    } catch (e) {
      console.log(e);
    }
    return { hello:  "world" };
  }
}

```

```
TypeError: Cannot read property 'get' of undefined
    at StatusController.get [as cache] (/home/ferco/workspace/http-service-template/lib/decorators/inject.ts:26:42)
    at StatusController.<anonymous> (/home/ferco/workspace/http-service-template/src/controllers/status.controller.ts:35:24)
    at Generator.next (<anonymous>)
    at /home/ferco/workspace/http-service-template/src/controllers/status.controller.ts:17:71
    at new Promise (<anonymous>)
    at __awaiter (/home/ferco/workspace/http-service-template/src/controllers/status.controller.ts:13:12)
    at StatusController.handler (/home/ferco/workspace/http-service-template/src/controllers/status.controller.ts:41:16)
    at Object.<anonymous> (/home/ferco/workspace/http-service-template/lib/decorators/strategies/controller-type.ts:52:61)
    at preHandlerCallback (/home/ferco/workspace/http-service-template/node_modules/fastify/lib/handleRequest.js:120:28)
    at preValidationCallback (/home/ferco/workspace/http-service-template/node_modules/fastify/lib/handleRequest.js:103:5)
```

**Comment by L2jLiga** [comment]
Hi there,

Does `AnyService` have `@Service` decorator?

**Comment by ferco0** [comment]
yes

**Comment by ferco0** [comment]
Yes. Works well injecting by controller constructor, but only with server started, in test environment using `configureControllerTest` cannot be mockable, `this.anyService` gets `undefined`
```javascript
constructor(private anyService: AnyService) {}
```

```javascript
instance = await configureControllerTest({
      controller: HelloController,
      mocks: [
        {
          provide: AnyService,
          useValue: { foo: "bar" },
        },
      ],
    });
```
```javascript
console.log(this.anyService); // undefined
```

**Comment by L2jLiga** [comment]
Thanks for clarification! I will take a look what I can do on it

**Comment by L2jLiga** [comment]
Released as [v3.1.1](https://github.com/L2jLiga/fastify-decorators/releases/tag/v3.1.1)

**Comment by ferco0** [comment]
Thanks!
